extensions/binding/org.eclipse.smarthome.binding.dmx/ESH-INF/thing/artnet-bridge.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="dmx"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+    xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+    <!-- ArtNet Bridge -->
+    <bridge-type id="artnet-bridge">
+        <label>ArtNet Bridge</label>
+        <description>The ArtNet bridge represents a single DMX universe connected via ArtNet, only unicast</description>
+
+        <channels>
+            <channel id="mute" typeId="mute"/>        
+        </channels>
+
+        <config-description>
+            <parameter name="address" type="text">
```

Comment:
Hm, I am not sure if I can follow you here. Where and how is it possible to give multiple addresses? The description says "Network address of bridge", so we are clearly speaking about a single one here. And as you have a default port, we would not require the user to always specify it.
Note that with the context "network-address" we would plan UI support in future (for syntax checking, possibly making suggestions, etc.), that's we it is strongly preferred to do it the same way in all bindings.